### PR TITLE
make versioninfo.py is suit for Python3.x

### PR DESCRIPTION
**Pull request by Cheain**
make versioninfo.py is suit for Python3.x. make command
'pyi-grab_version' is useful for Python3.x

### CODE REVIEW

File: PyInstaller/utils/win32/versioninfo.py [hunk 1]
```diff
@@ -337,27 +347,25 @@ def fromRaw(self, sublen, vallen, name, data, i, limit):
...
         sublen = 6 + 2*len(self.name) + 2
-        pad = ''
+        pad = b''
         if sublen % 4:
-            pad = '\000\000'
-        tmp = ''.join([kid.toRaw() for kid in self.kids])
+            pad = b'\000\000'
+        tmp = b''.join([kid.toRaw() for kid in self.kids])
         sublen = sublen + len(pad) + len(tmp)
         if tmp[-2:] == '\000\000':
```
Comment (poptosic) [inline PyInstaller/utils/win32/versioninfo.py]: Should this be `if tmp[-2:] == b'\000\000'`?

**Comment by htgoebel** [comment]
Thanks for this patch. I'm afraid it will break on Python 2.7 now. Have you tried running your code with Python 2.7?

I reviewed the code pointing out places which I assume will break in Python 2.7.

BTW: Did you know that the `U'…'` syntax is accepted by Python 3.3+ - which is the lowest version we support.

**Comment by htgoebel** [comment]
Are you going to update this?

**Comment by htgoebel** [comment]
I'm closing this pull-request in favor of #2623. Here questions have not been answered, unrelated changes are included and the code over-all looks not so clean.

**Pull request closed without merging**